armi/bookkeeping/report/reportingUtils.py

```diff
@@ -238,7 +238,7 @@ def getInterfaceStackSummary(o):
 
 
 def writeTightCouplingConvergenceSummary(convergenceSummary):
-    runLog.info("Tight Coupling Convergence Summary: Norm Type = Inf")
+    runLog.info("Tight Coupling Convergence Summary")
```

Comment:
at some point we shoudl add what is converged to the table, maybe a (c) value for when its converged

Reply:
doesn't have to be this PR

Reply:
Will address this with issue #1089.